e2e/src/helper.ts

```diff
@@ -139,16 +139,20 @@ export async function waitManifestJson(projectFolderName: string): Promise<void>
   const projectPath = getAbsolutePath(projectFolderName);
   if (fs.existsSync(path.resolve(projectPath, 'target', 'manifest.json'))) {
     console.log('manifest.json already exists');
+    console.log('Wait for manifest.json parse');
+    await sleep(200);
```

Comment:
Do you have an explanation why we need to wait if the file exists?

Reply:
Yes, after `manifest.json` created we need to wait when it parsed.

Reply:
Can you also combine logs here? I meant that here manifest.json is already exist and it is not clear to me why we need to wait any time here.